core/cat/memory/working_memory.py

```diff
@@ -34,6 +34,55 @@ class WorkingMemory(BaseModelDict):
 
     # track models usage
     model_interactions: List[ModelInteraction] = []
+    
+    def stringify_chat_history(self, latest_n: int = 5) -> str:
+        """Serialize chat history.
+        Converts to text the recent conversation turns.
+
+        Parameters
+        ----------
+        latest_n : int
+            Hoe many latest turns to stringify.
+
+        Returns
+        -------
+        history : str
+            String with recent conversation turns.
+
+        Notes
+        -----
+        Such context is placed in the `agent_prompt_suffix` in the place held by {chat_history}.
+
+        The chat history is a dictionary with keys::
+            'who': the name of who said the utterance;
+            'message': the utterance.
+
+        """
+
+        history = self.working_memory.history[-latest_n:]
+
+        history_string = ""
+        for turn in history:
+            history_string += f"\n - {turn['who']}: {turn['message']}"
+
+        return history_string
+
+    def langchainfy_chat_history(self, latest_n: int = 5) -> List[BaseMessage]:
+        chat_history = self.working_memory.history[-latest_n:]
```

Comment:
As above

Reply:
Yes, I missed it. I'm going to fix it now. 
For the imports in codebase, I checked in which files the methods were called but I didn't see a direct import to the old file stray_cat where they were, so I didn't edit it. Maybe I didn't read it correctly ?